ecllibrary/std/File.ecl

```diff
@@ -472,19 +474,20 @@ EXPORT SprayVariable(varstring sourceIP, varstring sourcePath, integer4 sourceMa
  * @param expireDays    Number of days to auto-remove file. Default is -1, not expire.
  * @param dfuServerQueue Name of target DFU Server queue. Default is '' (empty) for the first DFU queue in the environment.
  * @param noSplit      Don't split a file part to multiple target parts. Default is FALSE.
+ * @param keepSourceEncoding Prevent target file UTF8 encoding. Defaults to TRUE.
```

Comment:
>Prevent target file UTF8 encoding

I think better wording would be:

"If disabled, variable imports will default to UTF8. Defaults to TRUE."

Reply:
Changed